pulsar-client-cpp/tests/ConsumerTest.cc

```diff
@@ -797,4 +797,68 @@ TEST(ConsumerTest, testGetLastMessageIdBlockWhenConnectionDisconnected) {
     ASSERT_GE(elapsed.seconds(), operationTimeout);
 }
 
+class ConsumerSeekTest : public ::testing::TestWithParam<bool> {
+   public:
+    void SetUp() override { producerConf_ = ProducerConfiguration().setBatchingEnabled(GetParam()); }
+
+    void TearDown() override { client_.close(); }
+
+   protected:
+    Client client_{lookupUrl};
+    ProducerConfiguration producerConf_;
+};
+
+TEST_P(ConsumerSeekTest, testSeekForMessageId) {
+    Client client(lookupUrl);
+    auto n = std::chrono::system_clock::now();
+    auto now = std::chrono::duration_cast<std::chrono::nanoseconds>(n.time_since_epoch());
+
+    const std::string topic = "test-seek-for-message-id-" + std::to_string(now.count());
```

Comment:
We can simply use `std::to_string(time(nullptr))` as the topic suffix like other tests.